lxd/resource_lxd_container.go

```diff
@@ -309,6 +309,9 @@ func resourceLxdContainerExists(d *schema.ResourceData, meta interface{}) (exist
 	exists = false
 
 	ct, err := client.ContainerState(name)
+	if err != nil && err.Error() == "not found" {
+		err = nil
```

Comment:
The AWS one doesn't even have an exists function, so maybe that's why in there, they do this on Read:
```
func resourceAwsInstanceRead(d *schema.ResourceData, meta interface{}) error {
   conn := meta.(*AWSClient).ec2conn

   resp, err := conn.DescribeInstances(&ec2.DescribeInstancesInput{
      InstanceIds: []*string{aws.String(d.Id())},
   })
   if err != nil {
      // If the instance was not found, return nil so that we can show
      // that the instance is gone.
      if ec2err, ok := err.(awserr.Error); ok && ec2err.Code() == "InvalidInstanceID.NotFound" {
         d.SetId("")
         return nil
      }

      // Some other error, report it
      return err
   }
```